packages/shared/src/components/cards/MarketingCTA/MarketingCTACard.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import { Card } from '../Card';
+import { CardCover } from '../common/CardCover';
+import { CTAButton, Description, Header, MarketingCTA, Title } from './common';
+import { Card as CardV1 } from '../v1/Card';
+import { useFeedLayout } from '../../../hooks';
+
+export function MarketingCTACard({
+  marketingCTA,
+}: {
+  marketingCTA: MarketingCTA;
+}): ReactElement {
+  const { shouldUseMobileFeedLayout } = useFeedLayout();
+  const CardComponent = shouldUseMobileFeedLayout ? CardV1 : Card;
+
+  return (
+    <CardComponent className="p-4">
+      {marketingCTA.tagColor && marketingCTA.tagText && (
```

Comment:
Why not destruct marketingCTA?
`const { tagColor, tagText, title, description, image .... etc} = marketingCTA;`

Reply:
Can do, initially wanted to potentially drill it down, but let me refactor this :)